stdlib/websub/src/main/ballerina/src/websub/subscriber_client.bal

```diff
@@ -109,8 +114,8 @@ function buildSubscriptionChangeRequest(@untainted string mode,
 # + subscriptionChangeRequest - The subscription change request sent
 # + response - The http:Response or error received on request to the hub
 # + httpClient - The underlying HTTP Client Endpoint
-# + return - `SubscriptionChangeResponse` indicating subscription/unsubscription details, if the request was successful
-#            else `error` if an error occurred
+# + return - The subscription/unsubscription details if the request was successful or else an `error`
+#            if an error occurred
```

Comment:
```suggestion
#            if an error occurred
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.